JavaIntegration/src/main/java/org/openzen/zencode/java/module/JavaNativeTypeTemplate.java

```diff
@@ -138,13 +141,16 @@ private void loadMethods() {
 		for (Method method : class_.cls.getMethods()) {
 			if (isNotAccessible(method) || isOverridden(class_.cls, method))
 				continue;
+
 			if (expansion && !JavaModifiers.isStatic(method.getModifiers()))
 				continue;
 
-			Collection<MethodID> ids = new LinkedList<>();
-			boolean isStaticExpansion = false;
+			Collection<MethodID> ids = new LinkedHashSet<>(); // LinkedHashSet to prevent collision if a static method has both @Global and @Method
+			boolean isStaticExpansionMethod = expansion && method.isAnnotationPresent(ZenCodeType.StaticExpansionMethod.class);
+			boolean isStaticMethodInsideZc = (!expansion || isStaticExpansionMethod) && JavaModifiers.isStatic(method.getModifiers());
+
 			boolean implicit = false;
```

Comment:
💭 Thought:   
In theory, having `@Caster(implicit=true)` and `@Method` on a method would also make the method implicit, but since it was that way before, I guess that shouldn't be a problem for now...